fp1=open('data.txt','x')#file exist error



fp1.close()
